backend/src/models/services/account.ts

```diff
@@ -0,0 +1,37 @@
+import { Config, Journal, User } from '../index.js';
+import { ConfigType } from '../config.js';
+import { HydratedDocument } from 'mongoose';
+import { UserType } from '../user.js';
+
+/**
+ * 
+ * @param journalId 
+ * @returns 
+ */
+export async function getAccount(
+  journalId: string,
+) {
+  const journal = await Journal.findById(journalId);
+  if (!journal) throw new Error('Journal not found.');
+
+  const user = await User.findById(journal.user);
+  if (!user) throw new Error('User not found.');
+
+  // Journal config is optional
+  const config = await Config.findById(journal.config);
+
+  const result: {
+    config: HydratedDocument<ConfigType> | null,
+    user: HydratedDocument<UserType>,
+    setConfigMessage?: string
```

Comment:
I like this config message. I would just change the name because `setConfigMessage` is telling me that I can assign something to it instead of get it. The set and get verbiage is used mostly for classes.

```suggestion
    configMessage?: string
```